addons/account-info/addon.json

```
@@ -0,0 +1,30 @@
+{
+    "name": "Account Information",
+    "description": "Adds an info tab on the account settings page. Where you can see all of the information about your account.",
+    "credits": [
+      {
+        "name": "stio_studio",
+        "link": "https://scratch.mit.edu/users/stio_studio/"
+      }
+    ],
+    "dynamicEnable": true,
+    "dynamicDisable": true,
+    "userscripts": [
+      {
+        "url": "info.js",
+        "matches": ["https://scratch.mit.edu/accounts/settings/"]
+      },
+      {
+        "url": "info.js",
+        "matches": ["https://scratch.mit.edu/accounts/password_change/"]
+      },
+      {
+        "url": "info.js",
+        "matches": ["https://scratch.mit.edu/accounts/email_change/"]
+      }
```

Since these are arrays, these can be combined into one.
```suggestion
      {
        "url": "info.js",
        "matches": ["https://scratch.mit.edu/accounts/settings/", "https://scratch.mit.edu/accounts/password_change/", "https://scratch.mit.edu/accounts/email_change/"]
      }
```

---

That could be simplified even more.
```suggestion
      {
        "url": "info.js",
        "matches": ["https://scratch.mit.edu/accounts/*"]
      }
```